src/extensionApi.ts

```diff
@@ -225,7 +225,7 @@ export class CommandHandler {
     async downloadRuntime(): Promise<Protocol.Status> {
         const rtId: string = await this.promptDownloadableRuntimes();
         if (!rtId) {
-            return Promise.reject('Canceled by user');
```

Comment:
Command should return 'null | undefined' when cancelled. No need to reject it.

Reply:
if the method is async, isn't a simple ```return``` resulting in the same?